File: st.java
/**
 * st
 */
public class st {

    public static void main(String[] args) {
        int x = 10;
        int y = 2;
        int ans = divied(x,y);
        System.out.println(ans);
    }

    private static int divied(int x, int y){
        int count = 0;
        if (x%y == 0) {
            while (x - y != 0) {
                x = x - y;
                count+=1;
            }            
        }
        return count;
    }

}

File: st.js
/**
 * st
 */
// Function to divide two numbers
function divide(x, y) {
    let count = 0;  // Initialize count

    // Check if x is divisible by y
    if (x % y === 0) {
        while (x - y !== 0) {
            x = x - y;  // Subtract y from x
            count += 1;  // Increment the count
        }
    }

    return count;
}

// Main code
let x = 10;
let y = 2;
let ans = divide(x, y);

console.log(ans);  // This will print 4 to the console